app/src/ui/business-card/BusinessCard.types.ts

```diff
@@ -0,0 +1,20 @@
+import { ReactNode, CSSProperties } from "react";
+
+interface BusinessCardData {
```

Comment:
```suggestion
type BusinessCardData = {
```

Let's use `type`s intead of interfaces. 


Let's leave `interfaces` for Classes.